.dockerignore

```diff
@@ -1,6 +1,5 @@
 .git
 .gitignore
-**/target/
```

Comment:
.dockerignore file should remain the same as previously. See below to understand the reasoning.